### PR TITLE
FInal checks for PT-PT Translation

### PR DESCRIPTION
**Pull request by luisfontes19**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #< insert number here >.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: Document-ptpt/0x12-V7-Code_quality_and_build_setting_requirements.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# V7: Code Quality and Build Setting Requirements
+# V7: Requisitos de Qualidade de Codigo e DistribuiçãoRequisitos de Qualidade de Codigo e Configuração de Distribuição
```
Comment (sushi2k) [inline Document-ptpt/0x12-V7-Code_quality_and_build_setting_requirements.md:1]:
```suggestion
# V7: Requisitos de Qualidade de Codigo e Distribuição
```

**Review by sushi2k — CHANGES_REQUESTED** [review]
One thing should be changed. Otherwise looks good to me :-)

File: Document-ptpt/0x12-V7-Code_quality_and_build_setting_requirements.md (hunk 1 above)
Comment (luisfontes19) [reply Document-ptpt/0x12-V7-Code_quality_and_build_setting_requirements.md:1]: nice catch

**Review by sushi2k — APPROVED** [review]
Awesome work @luisfontes19! Thanks for the support and the final touches in this PR! Apologies for the late response, but was a busy month....

**Pull request merged**